data/en/mbs_0253.json

```diff
@@ -57,7 +57,7 @@
                                 "description": "You can select the dates of the period you are reporting for, if the given dates are not appropriate."
                             }],
                             "questions": [{
-                                    "question": "Volume of potable water that was supplied to customers"
+                                    "question": "Total volume of potable water that was supplied to customers"
```

Comment:
Spec say: `Total volume of potable water that was supplied to customers (megalitres)`, advise you get clarification from business.

Reply:
ahh good point, i presumed this based on the intro page, will double check

Reply:
This is because the desired copy (including the word "Total") causes the content to wrap and our accordion breaks.

"Total" needs removing until the UI component is fixed.

<img width="662" alt="screen shot 2018-06-07 at 11 17 01" src="https://user-images.githubusercontent.com/356075/41093864-4ebd204a-6a44-11e8-98d5-2354079bf4a6.png">